src/pe_reports/report_generator.py

```diff
@@ -521,18 +527,17 @@ def main():
     logging.info("Generating Graphs")
 
     # Create output directory
-    subprocess.call(["mkdir", args["OUTPUT_DIRECTORY"]])
+    subprocess.call(["mkdir", args["OUTPUT_DIRECTORY"]])  # nosec
 
     # Connect to cyhy database
     db_creds_file = args["--db-creds-file"]
     try:
         db = db_from_config(db_creds_file)
-    except OSError:
-        logging.critical(
-            f"Database configuration file {db_creds_file} does not exist", exc_info=True
-        )
+    except FileNotFoundError as not_found:
+        logging.error("%s : Missing input data. No report generated.", not_found)
         print("")
-        return 1
+        return 0
```

Comment:
`0` should only be returned in an error-free state.

```suggestion
        return 1
```

Reply:
0bb568a

Reply:
That does not fix the problem here. This should still return a `1` to indicate an error state.